src/compile/miniconstants.rs

```diff
@@ -19,6 +19,8 @@ pub fn init_constant_table() -> HashMap<String, Uint256> {
         ("Address_ArbosTest", 105),
         ("Address_ArbOwner", 107),
         ("Address_ArbGasInfo", 108),
+        ("Address_ArbAggregator", 109),
+        // address 110 reserved for EthBridge functionality
```

Comment:
If we're reserving this, I think the idea was to make any real call to the L2 contract at that address revert

Reply:
Added code to do this.